components/main_menu/main_menu.tsx

```diff
@@ -147,6 +157,8 @@ export class MainMenu extends React.PureComponent<Props> {
 
         const someIntegrationEnabled = this.props.enableIncomingWebhooks || this.props.enableOutgoingWebhooks || this.props.enableCommands || this.props.enableOAuthServiceProvider || this.props.canManageSystemBots;
         const showIntegrations = !this.props.mobile && someIntegrationEnabled && this.props.canManageIntegrations;
+        const teamsLimitReached = this.props.usageDeltaTeams >= 0;
+        const createTeamRestricted = this.props.isCloud && this.props.isCloudFreeEnabled && (this.props.isFreeTrial || teamsLimitReached);
```

Comment:
Should this be **not** in a free trial? During the free trial they should not be restricted to create a team 
```suggestion
        const createTeamRestricted = this.props.isCloud && this.props.isCloudFreeEnabled && (!this.props.isFreeTrial || teamsLimitReached);
```

Reply:
Ah ok, makes sense, thanks for clarifying 👍